src/autodiff.jl

```diff
@@ -7,10 +7,12 @@ using CUDA
 import CUDA.CUSPARSE
 import ForwardDiff
 import SparseDiffTools
+using KernelAbstractions
 
 using ..ExaPF: Spmat, xzeros, State, Control
 
 import Base: show
+KA = KernelAbstractions
```

Comment:
I would change this line as:
`const KA = KernelAbstractions`

Reply:
Done.